aws/components/adaptor/state.ftl

```diff
@@ -0,0 +1,28 @@
+[#ftl]
+
+[#-- Resources --]
+[#assign COT_ADAPTOR_RESOURCE_TYPE = "adaptor"]
+
+[#macro aws_adaptor_cf_state occurrence parent={} ]
+    [#local core = occurrence.Core]
+    [#local solution = occurrence.Configuration.Solution]
+
+    [#local id = formatResourceId(COT_ADAPTOR_RESOURCE_TYPE, core.Id)]
+
+    [#-- Baseline component lookup --]
+    [#local baselineLinks = getBaselineLinks(occurrence, [ "OpsData" ] )]
```

Comment:
This code looks unused. Is it there on the basis it will likely come into play in the future?

Reply:
Yeah someone might want to access the baseline links if they want